packages/lib/models/Note.ts

```diff
@@ -1125,7 +1126,8 @@ export default class Note extends BaseItem {
 	}
 
 	public static getNaturalSortingCollator() {
-		return new Intl.Collator(undefined, { numeric: true, sensitivity: 'base' });
+		const collatorLocale = currentLocale().slice(0, 2);
+		return new Intl.Collator(collatorLocale, { numeric: true, sensitivity: 'base' });
```

Comment:
You've added the same code in 4 different places. Could you move all this to a function and call it from those 4 places please? A good place for the function could be lib/models/utils for example

Reply:
Sounds good. I will make these changes